packages/protocol/contracts/layer1/preconf/iface/IProtector.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.27;
+
+import { ISlasher } from "@eth-fabric/urc/ISlasher.sol";
+
+/// @title IProtector
+/// @custom:security-contact security@taiko.xyz
+interface IProtector is ISlasher {
```

Comment:
Should we rename `Protector` to `Overseer` so we can unify with our fair exchange solution?

Reply:
Entity-wise I see them as same entity. Or are there any implementation-wise difference between them?